core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -1290,6 +1290,9 @@ private Message processMessageInAdapter(MessageWrapper<M> messageWrapper, PipeLi
 						}
 						getListener().afterMessageProcessed(pipeLineResult, messageForAfterMessageProcessed, session);
 					} catch (Exception e) {
+						itx.setRollbackOnly();
+						messageInError = true;
+						pipeLineResult.setState(ExitState.ERROR);
```

Comment:
Woops!!? This is from 'my' issue!

Reply:
Oops you're right. Another thing that leaked in 😳 Forgot to clean the workspace before doing the ports.